webrtc-stats.html

```diff
@@ -596,6 +586,10 @@ <h3>
               It is accessed by the <code><a>RTCIceCandidateStats</a></code> for the remote
               candidate.
             </p>
+            <p>
+              A remote candidate is deleted when the RTCIceTransport does an ICE restart, and the
+              candidate is no longer in use.
+            </p>
```

Comment:
the candidates are deleted when they are "no longer in use", which afaik is the case after ICE completed and chose an active pair? so, if one queries for stats e.g. only at the end of the connection, then it will never know what other candidates had been available. is this intended?

Reply:
i just read the definition of "statslifetimeended" event from webrtc-pc, so i guess my question is void. it will simply emit the event when the candidate(pair) is deleted?

Reply:
How do we capture that we want a single event from an ICE restart with one RTCStatsReport containing all the deleted candidates?

Reply:
I think I captured that in guidelines.